src/platform/web/Platform.js

```diff
@@ -189,6 +190,7 @@ export class Platform {
                 this._disposables.track(disposable);
             }
         }
+        this._container.addEventListener("error", handleAvatarError, true);
```

Comment:
Can you add a closure to disposables to remove this again when dispose gets called? `adaptUIOnVisualViewportResize` has an example of this.